platform/wayland/os-compatibility.h

```diff
@@ -21,6 +21,8 @@
  * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
+ *
+ * SPDX-License-Identifier: MIT
```

Comment:
Should we modify imported files from other projects as-is?

Reply:
Good point, maybe it's better to leave them be. I'll revert this change before landing.